components/date-picker/interface.tsx

```diff
@@ -68,10 +72,13 @@ export type RangePickerValue =
 export type RangePickerPresetRange = RangePickerValue | (() => RangePickerValue);
 
 export interface RangePickerProps extends PickerProps {
+  localeCode?: string;
```

Comment:
Is `localeCode` a RangePicker Prop?

Reply:
From code level seems yes, in `renderRangePicker` it gets `localCode` from `this.props`

Reply:
Hi @afc163 , I have to add @ts-ignore here to avoid typescript complain. But you are right, this `localeCode` does not work at all, then do you know why there's `fixLocale(value, localeCode)` in render?

Reply:
- `localCode` might be compatible with previous version and it's not recommended to use it.

Reply:
compatible？incompatible?